## 1.0 Data Cleaning

### Description:

The purpose of this step is to clean the data for the project. I will perform data cleaning tasks such as handling missing values, removing duplicates, and transforming data. Also, I will do feature engineering to improve the quality of the data and make it more suitable for analysis.

## 1.1 Import  libraries

Import the libraries needed for data cleaning and analysis

In [1]:
# base libraries
import pandas as pd
import numpy as np
import matplotlib.pyplot as plt
import seaborn as sns


## 1.2 Load data

The data for this project was downloaded from UCI Machine Learning Repository. The url is: https://archive.ics.uci.edu/dataset/697/predict+students+dropout+and+academic+success. The data was downloaded and stored in `data/raw/data.csv`.

In [5]:
data = pd.read_csv('../data/raw/data.csv', sep=';')

data.shape

(4424, 37)

***Interpretation:*** The dataset was loaded successfully. This dataset contains 4,424 rows and 37 columns. This indicates that there are 4,424 students in the dataset and 37 features for each student.

## Data preparation or Feature Engineering

## Analysis of Results and Conclusions

Description of the results obtained and if there are conclusions that can be drawn from them.

The analysis of results must be related to the description of the task.

**Note:** An analysis of results does not necessarily lead to conclusions, but to ideas or proposals for future work


## Proposals and Ideas

From the results obtained, what ideas or proposals can be generated to continue with the project
